### PR TITLE
Slovak translation

### PR DESCRIPTION
**Pull request by CWKevo**

### CODE REVIEW

File: languages/sk.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		'Enter the e-mail address:': 'Vložte E-mailovú adresu:',
+		'Enter the displayed text:': 'Vložte zobrazovaný text:',
+		'Enter URL:': 'Vložte URL adresu:',
+		'Enter the YouTube video URL or ID:': 'Vložte URL adresu YouTube videa alebo jeho ID:',
+		'Insert a Quote': 'Vložiť citát',
+		'Invalid YouTube video': 'Neplatné YouTube video',
+		'Left-to-Right': 'Zľava doprava',
+		'Right-to-Left': 'Zprava doľava',
+		'Drop files here': 'Presuňte súbory sem',
+
```
Comment (samclarke) [inline languages/sk.js:72]: There's some extra white space here that's causing the linter to complain, could you remove it? Also if you could add an empty line to the end as it complains if there isn't one.

**Comment by samclarke** [comment]
Thanks for translating!

There's some extra white space which is causing the linter to complain. If you could remove it that would be really helpful, it can then be merged without the linter complaining.

**Comment by CWKevo** [comment]
I don't know which one you meant, so I have removed all. There is already a whitespace at the end of the file.

FYI I have tested this locally using jsdelivr and this commit, it worked without issues

**Comment by live627** [comment]
> 
> 
> I don't know which one you meant, so I have removed most of them, including comments (they shouldn't be needed anyways, except the top one). There was already a whitespace at the end of the file.
> 
> FYI I have tested this locally using jsdelivr and this commit, it worked without issues

it was on line 72

**Comment by CWKevo** [comment]
> it was on line 72

Yes, all gone

**Comment by live627** [comment]
Thanks for the contribution by the way 👍

**Comment by CWKevo** [comment]
No problem, I just wanted to contribute to this amazing editor! I like the extensibility of it. It didn't have Slovak, so I decided to contribute. Ping me if something updates and you need to rephrase some strings

**Comment by samclarke** [comment]
Looks good. Merged, thanks!

**Pull request merged**